src/main/java/net/dries007/tfc/world/feature/vein/VeinFeature.java

```diff
@@ -105,7 +105,7 @@ protected void place(ISeedReader world, Random random, int blockX, int blockZ, V
     }
 
     @Nullable
-    protected BlockState getStateToGenerate(BlockState stoneState, Random random, C config)
+    protected BlockState getStateToGenerate(BlockState stoneState, Random random, C config, ISeedReader world, BlockPos pos)
```

Comment:
Minor nitpick, but almost everywhere else parameters are in the order world, pos, state. (IDEA can refactor this with a single dropdown menu), it's a bit weird to stick them on the end (given it's not a binary compat issue)